src/content/en/updates/2016/10/resizeobserver.md

```diff
@@ -0,0 +1,169 @@
+`getBoundingClientRect` or `getComputerStyle`, which can cause layout thrashing
+if you don’t take care of batching *all* your reads and *all* your writes.
+
+And then you realize that this doesn’t even cover the cases where elements
+change their size without main window having been resized. Appending new
+children, an element’s `display` style being set to `none` or similar actions
+can change the size of an element, its siblings or ancestors.
+
+This is why `ResizeObserver` is a useful primitive. They react to changes in
+size of any of the observed *elements*, independent on what caused that change.
+They provide you access to the new size of the observed elements, too. Let’s get
+straight into it!
+
+## API
+
+All the APIs with the “observer” suffix I mentioned above share a simple API
+design. `ResizeObserver` will fit in just fine. You create a `ResizeObserver`
+object and pass a callback to the constructor. The callback will be given an
+array of `ResizeOberverEntries` – one entry per observed element – which
+containthe new dimensions for the element.
+
+    var ro = new ResizeObserver( entries => {
+      for (let entry of entries) {
+        const cr = entry.contentRect;
+        console.log('Element:', entry.target);
+        console.log(`Element size: ${cr.width}px x ${cr.height}px`);
+        console.log(`Element padding: ${cr.top}px ; ${cr.left}px`);
+      }
+    });
+
+    // Observe one or multiple elements
+    ro.observe(someElement);
+
+## Some details
+
+### What is being reported?
+
+It is important to note that `ResizeObserver` *report* both the dimensions of
+the `contentRect` and the padding, while they only *watch* the `contentRect`.
+The `contentRect` is *not* to be confused with the bounding box of the element.
+Consequently, the top and left attribute of the `contentRect` are the offset
+from the border box. SVGs are an exception to the rule, where the dimensions of
+the bounding box will be reported.
+
+<img src="/web/updates/images/2016/10/resizeobserver/contentbox.png">
+
+### When is it being reported?
+
+The spec prescribes `ResizeObserver` to process all the resize events right
+before paint and after layout. This makes the callback of an `ResizeObserver`
+the ideal place to make changes to your page’s layout. Because `ResizeObserver`
+processing happens between layout and paint, doing so will only invalidate
+layout, not paint.
+
+### Gotcha
+
+You might be asking yourself: What happens if I change the size of an observed
+element inside the `ResizeObserver`’s callback? The answer is: You will trigger
+another call to the callback right away. However, `ResizeObserver` have a
+mechanism to avoid infinite callback loops and cyclic dependencies. Changes that
+cause additional `ResizeObserver` calls will only be processed in the same frame
+if the affected elements are deeper in the DOM tree than the *shallowest*
+element processed in the previous iteration of `ResizeObserver` entries.
```

Comment:
In other words, resize events propagate up the DOM tree a frame at a time. If that statement's true you might tack it to the end of this paragraph. If it's false you should explain why.